src/ui/components/SecondaryToolbox/redux-devtools/redux-annotations.ts

```diff
@@ -58,6 +58,12 @@ export const createDevtoolsAction = <S, A extends Action<unknown>>(
     action.request.instanceId = instanceId;
   }
 
+  // Crank up the max actions the RDT logic will process before compacting old ones
+  // This is usually 50 by default, but hey, let's be generous.  (We'll see if this
+  // ends up causing any memory issues or anything)
+  // @ts-ignore
+  request.maxAge = 500;
```

Comment:
What happens when things are "compacted" ? Will it cause runtime errors or just prevent users from interacting with the "compacted" ones ?

Reply:
Prevents the user from seeing the old ones.

When the `maxAge + 1`th action is dispatched, the `instances` reducer calculates the next state as usual, but consolidates the oldest action to effectively remove it from the selectable list.